packages/functional-tests/tests/oauth/oauthPromptNone.spec.ts

```diff
@@ -8,8 +8,12 @@ let email;
 const password = 'passwordzxcv';
 
 test.describe('oauth prompt none', () => {
-  test.beforeEach(async ({ pages: { login } }) => {
+  test.beforeEach(async ({ pages: { login } }, { project }) => {
     test.slow();
+    test.skip(
+      project.name === 'production',
+      'test plan not yet available in prod'
+    );
```

Comment:
Q: Is the plan to run these on production in a future train once stuff has landed?
Should this (and L149-152 below) have a comment w/ Jira ticket number?